overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -8783,7 +8789,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§6Rutile§r is a §6Titanium§r precursor found in §6Ilmenite Ore§r and §6Bauxite Ore§r, which spawn uncommonly in the Overworld.\n\n§rLarger quantities of Ilmenite and Bauxite may be found on the Moon and other planets.",
+          "desc:8": "§6Rutile§r is a §6Titanium§r precursor found in §6Ilmenite Ore§r and §6Bauxite Ore§r, which spawn uncommonly in the §eOverworld§r.\n\n§rLarger quantities of §6Ilmenite§r and §6Bauxite§r may be found on the §eMoon§r and other planets.\n\n§6Ilmenite§r dust can be processed into §6Rutile§r in a §acompletely renewable§r process that yields §6Wrought Iron§r as a byproduct.",
```

Comment:
Completely Renewable might make the player think that it is infinitely usable. Maybe exclude.